src/borg/crypto/key.py

```diff
@@ -810,13 +804,18 @@ def extract_nonce(self, payload):
             raise IntegrityError('Manifest: Invalid encryption envelope')
         return 42
 
+    def init_ciphers(self, manifest_data=None):
+        if manifest_data is not None and manifest_data[0] != self.TYPE:
+            raise IntegrityError('Manifest: Invalid encryption envelope')
+
     def encrypt(self, chunk):
         data = self.compressor.compress(chunk)
         return b''.join([self.TYPE_STR, data])
```

Comment:
guess we maybe could adapt/reuse lowlevel.UNENCRYPTED "dummy cipher" here and in PlaintextKey.
or kill it if we don't want it for that.